Code/Framework/AzCore/AzCore/Debug/LocalFileEventLogger.cpp

```diff
@@ -79,11 +79,11 @@ namespace AZ::Debug
 
     bool EventLogReader::Next()
     {
-        size_t increment = AZ_SIZE_ALIGN_UP(sizeof(IEventLogger::EventHeader) + m_current->m_size, EventBoundary);
+        size_t increment = AZ_SIZE_ALIGN_UP(sizeof(Metrics::IEventLogger::EventHeader) + m_current->m_size, AZ::Metrics::EventBoundary);
```

Comment:
nit: One namespace is `Metrics` and the next is `AZ::Metrics`.  Can you omit the `AZ` from the second one?

Reply:
Yes I can do that.